packages/kbn-storybook/package.json

```diff
@@ -18,14 +18,14 @@
     "fast-glob": "2.2.7",
     "glob-watcher": "5.0.3",
     "jest-specific-snapshot": "2.0.0",
-    "jest-styled-components": "6.3.1",
+    "jest-styled-components": "^7.0.2",
     "mkdirp": "0.5.1",
     "mini-css-extract-plugin": "0.7.0",
     "normalize-path": "3.0.0",
     "react-docgen-typescript-loader": "3.1.0",
     "rxjs": "6.5.2",
     "serve-static": "1.14.1",
-    "styled-components": "^3",
+    "styled-components": "^5.1.0",
     "webpack": "^4.41.5"
   }
-}
```

Comment:
@patrykkopycinski would you mind installing the `editorconfig` plugin in your IDE? Settings like https://github.com/elastic/kibana/blob/master/.editorconfig#L13 should ideally be consistently applied across the project.

Reply:
Fixed, Thank you @spalger